scripts/tls_budget_check.py

```
@@ -0,0 +1,293 @@
+    )
+    assert profile is not None
+    print(f"=== {args.label}")
+    print(f"    root samples : {profile.root}")
+    print(f"    {TARGET:<13}: {profile.target} ({profile.share:.1f}%)  budget {args.budget:.1f}%")
+    print(f"    stats        : {parse_stats(stats)}")
+    if profile.callers:
+        print("    remaining callers:")
+        for sym, n in profile.callers.most_common(8):
+            print(f"      {n:6d}  {sym}")
+    if ok:
+        print(f"    VERDICT      : PASS")
```

Ruff (0.16.1)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)